instructure/canvas-android
# [MBL-18024][Teacher] SpeedGrader split view can be collapsed

Test plan:
1. Test on large tablet, to have split view on SpeedGrader
2. Press collapse button, grade panel should collapse
3. Press expand button, grade panel should expand

refs: MBL-18024
affects: Teacher
release note: SpeedGrader side panel can be collapsed.

## Checklist
